source/adapters/level_zero/command_buffer.cpp

```diff
@@ -86,15 +86,39 @@ graphs.
 
 */
 
+namespace {
+/// Checks the version of the level-zero driver.
+/// @param Context Execution context
+/// @param VersionMajor Major verion number to compare to.
+/// @param VersionMinor Minor verion number to compare to.
+/// @param VersionBuild Build verion number to compare to.
+/// @return true is the version of the driver is higher than or equal to the
+/// compared version
+bool CheckDriverVersion(ur_context_handle_t Context, uint32_t VersionMajor,
```

Comment:
I feel like this name is ambiguous. Should we clarify that we check for a more recent driver? Something like `IsDriverNewer` or similar.